wikipedia/README.md

```diff
@@ -81,9 +81,10 @@ This track accepts the following parameters with Rally 0.8.0+ using `--track-par
   - `initial_ingest_bulk_size` (default: 100)
 - Ingest Operations:
   - `ingest_bulk_size` (default: 100)
-  - `as_warmup_time_periods` (default: [600,600,600,600,600]) 
+  - `as_warmup_time_periods` (default: [600,600,600,600,600])
   - `as_time_periods` (default: [1800,1800,1800,1800,1800])
   - `as_ingest_clients` (default: [1,2,4,8,16])
+  - `as_target_throughputs` (default: [-1,-1,-1,-1,-1])
```

Comment:
maybe use `as_ingest_target_throughputs` to make it consistent with `as_search_target_throughputs`?

Reply:
Renamed:
`as_target_throughputs` to `as_ingest_target_throughputs`
`p_as_target_throughputs` to `p_as_ingest_target_throughputs`